lua/lspconfig/pasls.lua

```diff
@@ -0,0 +1,39 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.pasls = {
+  default_config = {
+    cmd = { 'pasls' },
+    filetypes = { 'pascal' },
+    root_dir = util.path.dirname,
```

Comment:
this should include at the very least `.git`

Reply:
added now